docs/Documentation/Development/VSCode/index.md

```diff
@@ -77,3 +77,31 @@ Now use the Remote-SSH extension in VS Code to `Connect to Host...` and use the
 
 This should open a new VS Code window that will connect to the compute node automatically. You may begin browsing your home directory and editing files in the VS Code window.
 
+## Jupyter Notebook in VS Code
+Before proceeding with this section, please read [Interactive Parallel Python with Jupyter](../Languages/Python/KestrelParallelPythonJupyter/pyEnvsAndLaunchingJobs.md) as this document will be using its process as a reference.
```

Comment:
I don't think this example is needed to show the basic process

Reply:
Understood! I have stripped out all of the notes about the Kestrel Parallel Python Jupyter for a minimum example. See 87b5cc5